developer-model-service/readme.md

```diff
@@ -0,0 +1,23 @@
+# Developer Model Service
```

Comment:
What is the rationale for using lower case `readme` for this markdown ?

Is that a backstage thingy I need to learn ?

Reply:
No reason in particular, I've updated it to use uppercase